### ISSUE TITLE
[Feature request] Sniff for alternative opening tags ?

### ISSUE THREAD

**Issue by jrfnl** [squizlabs/PHP_CodeSniffer#1063]
The `Generic.PHP.DisallowShortOpenTag` sniff checks for PHP short open tags, however there are also some alternative syntaxes which have been removed in PHP7:
```php
<% echo 'content'; %>
<%= 'content'; %>
<script language="php">
echo 'content';
</script>
```
Ref: http://php.net/manual/en/language.basic-syntax.phptags.php

Would there be interest in a sniff for this ?

And if so - should this be added to the `Generic.PHP.DisallowShortOpenTag` sniff or should a separate sniff be created ?
